src/settings/BatchGroupConfigurationSettings/BatchGroupConfigurationSettings.js

```diff
@@ -45,6 +49,24 @@ const BatchGroupConfigurationSettings = ({ mutator }) => {
   // eslint-disable-next-line react-hooks/exhaustive-deps
   []);
 
+  const fetchBatchVoucherExports = useCallback(
+    (id) => {
+      if (id) {
+        setBatchVoucherExports();
+
+        mutator.batchVoucherExports.GET({
+          params: {
+            query: `batchGroupId==${id} sortby end/sort.descending start/sort.descending`,
+          },
+        })
+          .then(setBatchVoucherExports)
+          .catch(() => setBatchVoucherExports([]));
```

Comment:
do you need this catch? you have https://github.com/folio-org/ui-invoice/pull/230/files#diff-46b2286a67da66c98e17c2e31233ff96R55

and as for initial state value component should be fine with undefined value

Reply:
removed it

Reply:
it wouldn't be fine since it's passed to the component. Undefined means that it's loading and we can show spinner, empty list just displays empty list and it means that loading process is finished.

Reply:
in case undefined is used for loading, `[]` is reasonable